src/gui/menu_list.hpp

```diff
@@ -0,0 +1,40 @@
+//  SuperTux -- List menu
+//  Copyright (C) 2021 Rami <rami.slicer@gmail.com>
+//
+//  This program is free software: you can redistribute it and/or modify
+//  it under the terms of the GNU General Public License as published by
+//  the Free Software Foundation, either version 3 of the License, or
+//  (at your option) any later version.
+//
+//  This program is distributed in the hope that it will be useful,
+//  but WITHOUT ANY WARRANTY; without even the implied warranty of
+//  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+//  GNU General Public License for more details.
+//
+//  You should have received a copy of the GNU General Public License
+//  along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+#ifndef HEADER_SUPERTUX_GUI_MENU_LIST_HPP
+#define HEADER_SUPERTUX_GUI_MENU_LIST_HPP
+
+#include "gui/menu.hpp"
+
+class ListMenu final : public Menu 
+{
+public:
+  ListMenu(const std::vector<std::string>& items, int* selected_);
+
```

Comment:
Probably my last comment - is that trailing underscore there for something?

Reply:
Yes I forgot to update that during PR whackamolle